api/src/main/java/jakarta/el/BeanNameELResolver.java

```diff
@@ -15,7 +15,7 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  */
 
-package javax.el;
```

Comment:
We should also update the copyright date.  If a single date, we switch to `initial-date, 2020`.  If there are already two dates, just update second date to 2020.

https://github.com/eclipse-ee4j/glassfish-copyright-plugin can be cloned, built with maven and then run locally in various ways.  

First change into a folder that you can "git clone https://github.com/eclipse-ee4j/glassfish-copyright-plugin" and maven build that.

Some examples:

1.  To update only files modified but not yet committed to git:
repo=~/.m2/repository/org/glassfish/copyright/glassfish-copyright-maven-plugin
v=`ls $repo | grep '^[1-9]' | tail -1`
for i in `git status  --porcelain --untracked-files=no | cut -d' ' -f3`; do java -cp $repo/$v/glassfish-copyright-maven-plugin-$v.jar org.glassfish.copyright.Copyright -r -c $i; done

2.  To generate a script that does the copyright date modification:

rm -f /tmp/script.sh
echo '#!/bin/sh' > /tmp/script.sh

repo=~/.m2/repository/org/glassfish/copyright/glassfish-copyright-maven-plugin
v=`ls $repo | grep '^[1-9]' | tail -1`
for i in `git status  --porcelain --untracked-files=no | cut -d' ' -f3`; do echo java -cp $repo/$v/glassfish-copyright-maven-plugin-$v.jar org.glassfish.copyright.Copyright -r -c $i >> /tmp/script.sh; done
chmod +x /tmp/script.sh

Then just run /tmp/script.sh

3.  The same as #2 but slightly different to generate the script from changes already committed:

git checkout sync_ee9
git checkout -b newbranch_sync_ee9
echo "un-commit the commit"
git reset --soft HEAD~;git reset
echo "now do all the same steps as #2"

rm -f /tmp/script.sh
echo '#!/bin/sh' > /tmp/script.sh
repo=~/.m2/repository/org/glassfish/copyright/glassfish-copyright-maven-plugin
v=`ls $repo | grep '^[1-9]' | tail -1`
for i in `git status  --porcelain --untracked-files=no | cut -d' ' -f3`; do echo java -cp $repo/$v/glassfish-copyright-maven-plugin-$v.jar org.glassfish.copyright.Copyright -r -c $i >> /tmp/script.sh; done
chmod +x /tmp/script.sh

echo "now stash and delete this new branch"
git stash
git checkout sync_ee9
echo "now run /tmp/script.sh and then save the copyright date modifications and push your change"